### DISCUSSION TITLE
4/17 (월) 서브스크립트

### DISCUSSION THREAD

**Discussion by Eunho0922** [Info-iOS/iOStudy#83] [General]
서브스크립트는 타입의 요소에 접근하는 단축 문법입니다. 서브스크립트는 별도의 설정자 또는 접근자 등의 메서드를 구현하지 않아도 인덱스를 통해 값을 설정하거나 가져올 수 있습니다. 이와 같이 서브스크립트가 사용되고 왜 쓰이는지를 공부해보겠습니다. 오늘도 파이팅!

**Comment by jjunhaa0211** [comment]
오늘 전 것 까지 전부 올리겠습니다!!
